tests/test_helpers.py

```diff
@@ -58,7 +58,7 @@
      ""
      ),
     (["helper", "u_milestone", "20sb_firstprojectum", "--index", "4",
-      "-s", "p","-t", "m", "--due_date", "2020-05-30"],
+      "-s", "p", "-t", "m", "--due_date", "2020-05-30"],
```

Comment:
please put the full varable names here to make it more readable as documentation.   Actually, some of the other helpers probably also need this treatment

Reply:
Done